composer.json

```
@@ -20,7 +20,7 @@
   "require-dev": {
     "phpstan/phpstan": "^1.10.5",
     "phpstan/phpstan-symfony": "^1.2.20",
-    "gotenberg/gotenberg-php": "^1.0.0",
+    "gotenberg/gotenberg-php": "^1.0|^2.0",
```

We should use double pipe consistently. Single pipe is a old alternative.
See https://getcomposer.org/doc/articles/versions.md#version-range
```suggestion
    "gotenberg/gotenberg-php": "^1.0 || ^2.0",
```